src/test/java/com/github/fishio/TestPlayerFish.java

```diff
@@ -92,9 +94,43 @@ public void testCollideWithLargerEnemyFish() {
 				null, 0.0, 0.0));
 		when(ef.getBoundingArea().getSize()).thenReturn(6.1);
 		
+		//Set the amount of lives to 1
+		pf.livesProperty().set(1);
+		
+		//Collide with an enemy
+		pf.onCollide(ef);
+		
+		//The player should have no more lives left.
+		assertEquals(0, pf.getLives());
+		
+		//The player should be dead.
+		assertTrue(pf.isDead());
+		
+		Mockito.verify(ef, never()).setDead();
```

Comment:
Is it necessary to do Mockito.verify on the setDead method? I mean, why would it be bad if the playerFish calls the setDead() method? The important part is that you do the assertTrue(pf.isDead()) check.

<layout